crates/ruff_linter/src/message/sarif.rs

```diff
@@ -153,6 +153,58 @@ struct SarifResult<'a> {
     start_column: OneIndexed,
     end_line: OneIndexed,
     end_column: OneIndexed,
+    fixes: Vec<SarifFix>,
+}
+
+#[derive(Debug, Serialize)]
```

Comment:
Unrelated to your PR: It would be great to include the link to `https://docs.oasis-open.org/sarif/sarif/v2.1.0/csprd01/sarif-v2.1.0-csprd01.html#_Toc10541319` somewhere in the documentation (on `SarifResult` or on `SarifEmitter`)